src/CMakeLists.txt

```diff
@@ -110,10 +110,21 @@ configure_file(../valhalla/valhalla.h.in ${CMAKE_CURRENT_BINARY_DIR}/valhalla/va
 configure_file(../valhalla/config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/valhalla/config.h @ONLY)
 
 set(valhalla_hdrs
-  ${CMAKE_CURRENT_BINARY_DIR}/valhalla/valhalla.h
-  ${VALHALLA_SOURCE_DIR}/valhalla/worker.h)
+    ${CMAKE_CURRENT_BINARY_DIR}/valhalla/valhalla.h
+    ${VALHALLA_SOURCE_DIR}/valhalla/worker.h)
+
+set(valhalla_src
+    worker.cc
+    ${CMAKE_CURRENT_BINARY_DIR}/valhalla/config.h
+    ${valhalla_hdrs}
+    ${libvalhalla_link_objects})
+
+if (ENABLE_SERVICES)
+  set(valahlla_hdrs ${valhalla_hdrs} ${VALHALLA_SOURCE_DIR}/valhalla/tile_server.h)
```

Comment:
is this deliberate (valahlla_hdrs)?

Reply:
It was just a simple way to have an array for all the headers so we could optionally add this one header which  doesnt make sense in the case that you dont have http enabled.